Jenkinsfile

```diff
@@ -301,7 +309,15 @@ def taskRunTests(cfg) {
 	cepl.cleanCurrentDir()
 	unstash(name: SOURCE_STASH)
 
-	String buildCmd = "python -m pip install . && python tests/run_tests.py --xml_output_directory ${cfg.ws}"
+    String venv = "${cfg.ws}/venv"
+	String buildCmd = ''
+	if (isUnix()) {
+	    buildCmd = "cp -r /tmp/pyprt-build-venv ${venv}" // copy ro venv so we can install to run tests
+	    buildCmd += " && ${venv}/bin/python -m pip install . && ${venv}/bin/python tests/run_tests.py --xml_output_directory ${cfg.ws}"
+	}
+	else {
+	    buildCmd = "python -m pip install . && python tests/run_tests.py --xml_output_directory ${cfg.ws}"
```

Comment:
There seems to be a bit of code duplication in `taskBuildWheel/taskBuildConda/taskRunTests`. Could maybe be improved.

Reply:
True, though it is always slightly different... not sure it is easier to read after pulling out stuff... will leave it like this for now, sry.